README.md

```diff
@@ -8,7 +8,7 @@ See it in action at [ethercalc.net](http://ethercalc.net)!
 * jQuery (only on the client-side)
 
 ## Usage
-You can `require('socialcalc')` or include `node_modules/socialcalc/SocialCalc.js` on your web page using a script tag.
+You can `require('socialcalc')` or include `node_modules/socialcalc/dist/SocialCalc.js` on your web page using a script tag.
```

Comment:
This will require updating EtherCalc, but looks good otherwise.

Reply:
True, would break this line - https://github.com/audreyt/ethercalc/blob/master/Makefile#L50